### ISSUE TITLE
Move `[ap_hashtags]` last in post in Content template

### ISSUE THREAD

**Issue by aslakr** [Automattic/wordpress-activitypub#432]
Maybe `[ap_hashtags]` should be moved last in the default "Content" post template given how Mastodon 4.2 will treat last line hashtags? 

I.e. from:

https://github.com/Automattic/wordpress-activitypub/blob/f9218ebb1bb0632ec15995afdbb50d8e126b27d8/includes/transformer/class-post.php#L450

to

```php
 return "[ap_content]\n\n[ap_permalink type=\"html\"]\n\n[ap_hashtags]";
```

**Comment by pfefferle** [comment]
Isn't that the behaviour if you have no hashtags in your posts, but only as meta?

https://docs.joinmastodon.org/spec/activitypub/#Hashtag

**Comment by mediaformat** [comment]
@pfefferle I believe the out-of-band or meta hashtags are those not found in the Content but only in the `tag` array

**Comment by aslakr** [comment]
Yes, it seems like `[ap_hashtags]` may not be needed in the Mastodon 4.2+ web ui, which means that the Content section could be reduced to:

```php
 return "[ap_content]\n\n[ap_permalink type=\"html\"]";
```

but on other clients (and other activitypub implementations) those hashtags might not _yet_ be available the ui.

**Comment by mediaformat** [comment]
I would keep it, sorry got not providing more context in my last comment. 

Here is a quick test: The hashtags were places on the last line, and moved to the bottom automatically
<img width="592" alt="Screenshot 2023-09-22 at 3 47 19 PM" src="https://github.com/Automattic/wordpress-activitypub/assets/2895351/10e1c20c-f474-4f4e-bd62-9269fd2c8055">

Hashtags not on the last line, don't appear in the `HashTagBar`
<img width="592" alt="Screenshot 2023-09-22 at 3 52 42 PM" src="https://github.com/Automattic/wordpress-activitypub/assets/2895351/e9ce2fd3-8bbc-4f01-a676-f1e1f91fc210">

To summarize: Only Tags placed on the last line, or Out of Band will appear separately at the bottom. 
Regardless if we are returning a list of tags having them on the last line seems like good practice regardless of other platform UIs
